R/fct_tables.R

```diff
@@ -176,6 +176,23 @@ create_table.general <- function(
 }
 
 
+#' Create Default 'Common Events' Table
+#'
+#' @export
+#' @inherit create_table.default 
+create_table.common_events <- function(
```

Comment:
I think the function should be named `create_table.common_forms`, right?

Reply:
Oopsies